### DISCUSSION TITLE
Installation issue on CentOS 7 - 500 Internal Server Error

### DISCUSSION THREAD

**Discussion by redbinkieresearchlabs** [kimai/kimai#2727] [On-Premise: Installation & Updates] [answered]
Hi,

I am stuck at this screen for a fresh install.

Am running Apache on Centos 7. PHP and MySQL is installed.

DB connection is fine as i can see the tables are also created.

I believe it could be a file permission issue as there is no log file being generated. I have checked all permissions and they are looking correct.

Example:

drwxrwxrwx.  6 apache apache    160 Aug 24 06:21 assets
drwxrwxrwx.  2 apache apache     21 Aug 24 06:47 bin
-rwxrwxrwx.  1 apache apache   8136 Aug 24 06:21 CHANGELOG.md
-rwxrwxrwx.  1 apache apache   6320 Aug 24 06:21 composer.json
-rwxrwxrwx.  1 apache apache 542793 Aug 24 06:21 composer.lock
drwxrwxrwx.  6 apache apache    182 Aug 24 06:21 config

My webserver runs on the apache user. Running a cache rebuild also throws no errors, everything is green.

Have hit a wall now and not sure how to progress further. Any pointers on what could be wrong? Thanks so much.

Regards,
Asrar

**Comment by kevinpapst** [comment, marked as answer]
Replace `www-data` with `apache` in https://www.kimai.org/documentation/installation.html#file-permissions

EDIT: SELinux and PHP apps are often not the best friends in default CentOS setups. Try deactivating SELInux and test Kimai again.

**Reply by redbinkieresearchlabs** [reply]
Hi Kevin,

Yeah thats what I did. I restart the apache service after that. Didnt work so i restart my OS as well. Still stuck with this unfortunately.

**Reply by kevinpapst** [reply]
Maybe that is SELinux disallowing access. Try to deactivate it for testing.

**Reply by redbinkieresearchlabs** [reply]
That was perfect Kevin! It was the selinux that was blocking. Thank you so much for pointing in the right direction.
